plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/ValidateMediatorPropertiesEditionPartForm.java

```diff
@@ -960,39 +963,48 @@ protected Composite createSourceWidget(FormToolkit widgetFactory, final Composit
         widgetFactory.paintBordersFor(parent);
         GridData valueData = new GridData(GridData.FILL_HORIZONTAL);
         sourceText.setLayoutData(valueData);
-        sourceText.addFocusListener(new FocusAdapter() {
-            /**
-             * @see org.eclipse.swt.events.FocusAdapter#focusLost(org.eclipse.swt.events.FocusEvent)
-             * 
-             */
+
+        sourceText.addMouseListener(new MouseAdapter() {
+            
             @Override
-            @SuppressWarnings("synthetic-access")
-            public void focusLost(FocusEvent e) {
+            public void mouseDown( MouseEvent event ) {
+                openNamespacedPropertyEditor(parent);
             }
-
-            /**
-             * @see org.eclipse.swt.events.FocusAdapter#focusGained(org.eclipse.swt.events.FocusEvent)
-             */
+            
+        });
+        
+        sourceText.addKeyListener(new KeyListener() {
+                        
             @Override
-            public void focusGained(FocusEvent e) {
-                EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
-                        SWT.NULL, source);
-                // valueExpression.setPropertyValue(valueExpressionText.getText());
-                nspd.open();
-                sourceText.setText(source.getPropertyValue());
-                propertiesEditionComponent
-                        .firePropertiesChanged(new PropertiesEditionEvent(ValidateMediatorPropertiesEditionPartForm.this,
-                                EsbViewsRepository.ValidateMediator.Properties.source, PropertiesEditionEvent.COMMIT,
-                                PropertiesEditionEvent.SET, null, getSource()));
+            public void keyPressed(KeyEvent e) {
+                openNamespacedPropertyEditor(parent);
             }
+            
+            @Override
+            public void keyReleased(KeyEvent e) {}
+            
         });
+        
         EditingUtils.setID(sourceText, EsbViewsRepository.EnrichMediator.Source.sourceXPath);
         EditingUtils.setEEFtype(sourceText, "eef::Text");
         Control sourceXPathHelp = FormUtils.createHelpButton(widgetFactory, parent, propertiesEditionComponent
                 .getHelpContent(EsbViewsRepository.ValidateMediator.Properties.source, EsbViewsRepository.FORM_KIND),
                 null); // $NON-NLS-1$
         return parent;
     }
+    
+    private void openNamespacedPropertyEditor(final Composite parent) {
+        EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
+                SWT.NULL, source);
+        // valueExpression.setPropertyValue(valueExpressionText.getText());
```

Comment:
Remove commented line.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/commit/1d53e06b88e566a6d316666c06dd48202d396d30